packages/itwin/ec3-widget/src/components/templateClient.tsx

```diff
@@ -0,0 +1,121 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { Guid } from "@itwin/core-bentley";
+import { Template, Label, Material } from "./Template"
+
+// For now only one template for each report.
+
+export default class TemplateClient {
+  getTemplatesT(): Template[] {
```

Comment:
If the method is public - mark it as `public`.

Reply:
public keyword added, unnecessary methods removed